specs/architecture.md

```diff
@@ -55,7 +53,25 @@ User interface needs also some data visualizations of all the data so the [data2
 
 For the RESTful API endpoints generation in Android app the [Kotlin code generator](https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators/kotlin.md) is used.
 
-The client app is designed to be multilanguage app  to provide access to occupancy data also for foreign people. All the translations are managed via resource files with maapings for application text views.
+If needed custom widgets can be designed by developers team or used some open-source alternatives to standard Android widgets from web.
+
+The client app is designed to be multilingual app  to provide access to occupancy data also for foreign people. All the translations are managed via resource files with mappings for application text views.
+
+### App UI design
+
+Android app is divided in a few activities that allows users to work with different tasks and some of them consists of a few fragments. The initially planned architecture is to use activities [A] and fragments [F] for:
+1. Hello application screen to give the user possibility to choose if he would like to use list or map search [A]
+2. Search map of canteens data [A]
+3. List search with filters of canteens data [A]
+4. Settings screen of application [A]
+5. Main application screen with presented data [A] divided into:
+
+    * Main canteen data such as name, logo, opening hours, base stats from day, descriptions [F]
+
```

Comment:
Extra newlines? Markdown parser will understand these points as three separate lists.

Reply:
lets see to the "compiled version", it works in this way https://github.com/pooh-labs/eaterslab/blob/avan1235/spec-fix-arch/specs/architecture.md